docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/DropColumns.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class DropColumns
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable and them read it as ML.NET's data type.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleInfertData> data = SamplesUtils.DatasetUtils.GetInfertData();
+            var trainData = mlContext.Data.ReadFromEnumerable(data);
+
+            // Preview of the data.
+            //
+            // Age    Case  Education  Induced     Parity  Pooled.stratum  Row_num  ...
+            // 26.0   1.0   0-5yrs      1.0         6.0       3.0      1.0  ...
+            // 42.0   1.0   0-5yrs      1.0         1.0       1.0      2.0  ...
+            // 39.0   1.0   0-5yrs      2.0         6.0       4.0      3.0  ...
+            // 34.0   1.0   0-5yrs      2.0         4.0       2.0      4.0  ...
+            // 35.0   1.0   6-11yrs     1.0         3.0       32.0     5.0  ...
+
+            // DropColumns is commonly used to drop unused columns if the dataset is going to be serialized or
+            // written out to a file. It is not actually necessary to drop unused columns before training or
+            // performing transforms, as IDataView's lazy evaluation won't actually materialize those columns.
+            // In the case of serialization, every column in the schema will be written out. If you have columns
+            // that you don't want to save, you can use DropColumns to remove them from the schema.
```

Comment:
Does it make sense to add this info to the remarks section?

Reply:
Yes, moved to remarks.